### ISSUE TITLE
7.8.18 Content Field on Campaign module can't be edited on inline editing.

### ISSUE THREAD

**Issue by take4tuz** [salesagility/SuiteCRM#5948]
I can't do inline editing on  the "content" field on Campaign module.

I can start inline editing on that field, but  I can't finish it.
The check button that appear on the right side of the field doesn't seem to be working. 

I found that  id="content" appears twice in the same html file
One is for the "context" field on Campaign, another is for the theme.
I guess if the id="keyword" appear twice or more on the same html file, inline editing doesn't work. 

I tried to fix it.
The "content" field on Campaign is the build-in field, so it is difficult to change the field name.
So I tried to change theme files.

I  replaced every "content" on theme/SuiteP/*.css to "theme-content", and replaced id="content" to id="theme-content" in theme/SuiteP/*.tpl.
It works fine.

Basically, the field name on Campaign should be changed, but it it difficult to keep maintenance if I change it by myself.

[My environment]
* SuiteCRM Version used: 7.8.18
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Firefox 60.0.1
* Environment name and version (e.g. MySQL, PHP 7): MySQL PHP7.1
* Operating System and version (e.g Ubuntu 16.04): CentOS

**Comment by PedroErnst** [comment]
Hi @take4tuz 

Good work at researching the issue.

I think there is no straightforward solution for this. As you pointed out changing the name of the field in Campaigns is likely to break things. 

But equally, changing the id of such a central div is very likely to cause issues, there are plenty of references to this id not only in css, but also in js files. Plus whatever customizations out there rely on this.

I'm not sure if this approach is doable.

**Comment by take4tuz** [comment]
Thanks for your comment.
It is not critical but I hope It will be fixed someday soon.

**Comment by samus-aran** [comment]
To be honest this won't be fixed unless it is a major version and documentation to highlight this. We also have to consider do we want content to be inline-editable. We could apply inline_edit = false for this field to remove this functionality for a clear resolve but in your case you obviously use it but what would be the communitys' preference?

**Comment by pribeiro42** [comment]
Would it be possible to change the id of the main div instead of just removing the ability to edit this field in-line ?
